### PR TITLE
[OTA-1530] Verify displaying correct data

### PR DESCRIPTION
**Pull request by piotrsmol**
Signed-off-by: Piotr Smol <piotrsmol.pvt@gmail.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/components/updates/UpdateDetails.jsx [hunk 1]
```diff
@@ -3,19 +3,43 @@
...
 
+  componentDidUpdate() {
+    if (!this.fromVersion && !this.toVersion) {
+      const { updatesStore, softwareStore } = this.props.stores;
+      const mtuData = updatesStore && updatesStore.currentMtuData && updatesStore.currentMtuData.data;
+      _.each(mtuData, (target) => {
+        const { target: fromPackage } = target.from;
+        const { target: toPackage } = target.to;
+        _.find(softwareStore.packages, pack => {
+          const version = pack.id.version;
```
Comment (piotrcichonmee) [inline ota-plus-web/app/reactapp/src/components/updates/UpdateDetails.jsx]: Use destructuring

File: ota-plus-web/app/reactapp/src/components/updates/UpdateDetails.jsx [hunk 2]
```diff
@@ -47,9 +71,9 @@ class UpdateDetails extends Component {
           ) : mtuData ? (
             _.map(mtuData, (target, hardwareId) => {
               const noInformation = 'No information.';
-              const { target: fromPackage, checksum: fromVersion } = target.from;
-              const { target: toPackage, checksum: toVersion } = target.to;
-
+              const { target: fromPackage } = target.from;
+              const { target: toPackage } = target.to;
+
```
Comment (piotrcichonmee) [inline ota-plus-web/app/reactapp/src/components/updates/UpdateDetails.jsx]: Remove unnecessary space-signs.

File: ota-plus-web/app/reactapp/src/components/updates/UpdateDetails.jsx (hunk 1 above)
Comment (piotrsmol) [reply ota-plus-web/app/reactapp/src/components/updates/UpdateDetails.jsx]: done

File: ota-plus-web/app/reactapp/src/components/updates/UpdateDetails.jsx (hunk 2 above)
Comment (piotrsmol) [reply ota-plus-web/app/reactapp/src/components/updates/UpdateDetails.jsx]: done

**Pull request merged**